.eslintrc.json

```diff
@@ -30,7 +30,8 @@
     "settings": {
         "import/parsers": {
             "@typescript-eslint/parser": [".ts", ".tsx"]
-        }
+        },
+        "jest": { "version": 27 }
```

Comment:
We need this and `eslint-jest-plugin` because `eslint-config-preact` requires it :expressionless: 
https://github.com/preactjs/eslint-config-preact/issues/19#issuecomment-997924892